grails-app/assets/javascripts/projects.js

```diff
@@ -819,6 +827,9 @@ function ProjectViewModel(project, isUserEditor) {
         }
     }
 
+    self.transients.nespCategories = [ "Category 1: Indigenous-led", "Category 2: Co-design", "Category 3: Communicate", "N/A"];
```

Comment:
does look like`nespCategories` and  `nespNationalScaleLevels` are used. remove them.